bigbone/src/main/kotlin/social/bigbone/api/method/MediaMethods.kt

```diff
@@ -17,15 +19,20 @@ class MediaMethods(private val client: MastodonClient) {
      * Creates an attachment to be used with a new status. This method will return after the full sized media is done processing.
      * @param file the file that should be uploaded
      * @param mediaType media type of the file as a string, e.g. "image/png"
+     * @param focus Two floating points (x,y), comma-delimited, ranging from -1.0 to 1.0
```

Comment:
The parameter is a `Focus` object, not two comma-delimited values. That string gets built internally. :)

Reply:
Tried to modify it, but the companion object is static and I need to access instance member variables (x and y). Not sure if this is really doable... Could you share an example?

Reply:
> The parameter is a `Focus` object, not two comma-delimited values. That string gets built internally. :)

Fixed